PresContest/src/org/icpc/tools/presentation/contest/internal/ClientLauncher.java

```diff
@@ -110,8 +123,39 @@ public void showHelp() {
 			client = new PresentationClient(name, "!admin", cdsSource);
 		instance = client;
 
+		// load default presentation
+		Presentation presentation = null;
+		if (presStr[0] != null) {
+			Trace.trace(Trace.INFO, "Default presentation: " + presStr[0]);
+			List<PresentationInfo> presentations = PresentationHelper.getPresentations();
+			if (presentations.isEmpty()) {
+				Trace.trace(Trace.USER, "No presentations found");
+				System.exit(0);
+			}
+
+			PresentationInfo pi = StandaloneLauncher.findPresentation(presentations, presStr[0]);
+			if (pi == null)
```

Comment:
Maybe also add an trace here that the presentation doesn't exist?

Reply:
findPresentation() already traces and does a system exit if the presentation can't be found or matched so this code is unnecessary and could be removed, only kept it since it matches what the standalone client had.